ert_gui/simulation/detailed_progress.py

```diff
@@ -179,7 +165,7 @@ def _get_byte_with_unit(byte_count):
 
     def data(self, index, role):
         if not index.isValid():
-            return QVariant()
```

Comment:
Is returning an emtpy QVariant the same as returning None?

Reply:
I did some testing and it seems like returning None is the correct behaviour when using qtpy:

```python
from qtpy.QtCore import Qt, QStringListModel, QModelIndex
from qtpy.QtWidgets import QApplication


def main():
    app = QApplication([])
    model = QStringListModel([])
    print(model.data(QModelIndex(), Qt.DisplayRole) is None)


if __name__ == "__main__":
    main()
```
Here we're trying to access an empty string-list model using an invalid (empty) index, which returns `QVariant()` in C++. This prints `True` in both Py2+Qt4 and Py3+Qt5 because PyQt automatically converts to Python objects.

Thus, our overloaded `data` method also needs to return `None` and not `QVariant()` to be consistent with the behaviour of the rest of the program.

Reply:
:+1: